cmd/plasmacli/subcmd/eth/status.go

```diff
@@ -0,0 +1,45 @@
+package eth
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/cmd/plasmacli/config"
+	"github.com/FourthState/plasma-mvp-sidechain/eth"
+	"github.com/spf13/cobra"
+)
+// StatusCmd returns the current state of the eth connection (syncing, crashed etc)
+func StatusCmd() *cobra.Command {
+	return statuscmd
+}
+
+var statuscmd = &cobra.Command {
+	Use:   "status",
+	Short: "check state of eth connection",
+	Long: `returns current state of eth connection (syncing, crashed, etc)`,
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		// Get Conf
+		conf, err := config.ParseConfigFromViper()
+		if err != nil {
+			return fmt.Errorf("error retrieving config: { %s }", err)
```

Comment:
also, can you remove the `{` wrapper. Common go error format is "err: subser". I know this is inconsistent in other parts of the codebase but working towards fixing it

Reply:
same applies for errors below